### PR TITLE
providers/radius: simple radius outpost

### PR DESCRIPTION
**Pull request by BeryJu**
A simple radius outpost that only supports username/password authentication (using the same flow executor as the ldap outpost), no support for CHAP as that would require reversible password storage, no support for TLS as there are multiple ways to implement that but none are supported by the current go library

### CODE REVIEW

File: authentik/providers/radius/models.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+class RadiusProvider(OutpostModel, Provider):
+    """Allow applications to authenticate against authentik's users using Radius."""
+
+    shared_secret = models.TextField(
+        default=generate_key,
+        help_text=_("Shared secret between clients and server to hash packets."),
+    )
+
+    client_networks = models.TextField(
+        default="0.0.0.0/0",
```
Comment (samip5) [inline authentik/providers/radius/models.py:23]: Is that going to be both IPv6 and IPv4? or only IPv4?

**Comment by ferferga** [comment]
@BeryJu Care to explain why this has been closed? Was eagerly waiting for this :(

**Comment by BeryJu** [comment]
> @BeryJu Care to explain why this has been closed? Was eagerly waiting for this :(

I was just renaming some branches for cleanup, I forgot that closes the PR for it too.

Also this probably wont happen for a while since all radius implementations that use anything but the very basic features require access to unhashed passwords somewhere wihch I dont want to save.

**Comment by MrSuicideParrot** [comment]
@BeryJu  could you share with us what is the status of the Radius outpost? What are the plans of authentik regarding it?

I've spend a week trying to configure freeradius and authentik with the ldap outpost for a VPN with DUO, and realized that working with ldap is two limiting. Even though you are only implementing clear text passwords, I think that it would be better than having to rely on the ldap outpost.

**Comment by cwildfoerster** [comment]
Would love to see radius support too.

**Pull request merged**